docs/contributing-to-airbyte/monorepo-python-development.md

```diff
@@ -88,4 +88,12 @@ The following setup steps are written for PyCharm but should have similar equiva
 
 You should now have access to code completion and proper syntax highlighting for python projects.
 
-If you need to work on another connector you can quickly change the current virtual environment in the bottom toolbar.
\ No newline at end of file
+If you need to work on another connector you can quickly change the current virtual environment in the bottom toolbar.
+
+### Excluding files from venv
+
+By default, the find function in IntelliJ is not scoped and will include all files in the monorepo, including all the libraries installed as part of a connector's virtual environment. This huge volume of files makes indexing and search very slow. You can ignore files from the connectors' virtual environment with the following steps:
+
+1. Open the project structure using `cmd-;`
+2. Add the following filter to the `Exclude files` option: `connectors/**/.venv`
```

Comment:
It might be useful to say this is in the "Project Settings / Modules"  section

Reply:
might also be useful to add a screenshot

Reply:
added missing steps and a screenshot